lib/_debugger.js

```diff
@@ -1639,7 +1639,16 @@ Interface.prototype.trySpawn = function(cb) {
   } else if (this.args.length === 3) {
     // `node debug -p pid`
     if (this.args[1] === '-p' && /^\d+$/.test(this.args[2])) {
-      process._debugProcess(parseInt(this.args[2], 10));
+      const pid = parseInt(this.args[2], 10);
+      try {
+        process._debugProcess(pid);
+      } catch (e) {
+        if (e.code === 'ESRCH') {
+          console.error('Target process: ' + pid + ' doesn\'t exist.');
```

Comment:
Yes, We can use template string literals. We switched closure-linter to eslint. 
https://github.com/nodejs/io.js/blob/master/.eslintrc#L7